scripts/git-clang-format-custom

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/env python
+# Copy of https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format
```

Comment:
As it says, this is a customized version of the normal git-clang-format. Only used in ```clang-format.sh diffstat```. diffstat provides the list of files whose formatting is not ok. I use that for the github action as I find it much nicer than showing an actual diff.

Note the different license of this file as it's from clang as the rest of Suricata. I think it should be ok, as it applies to the whole file?

Reply:
yeah I don't see an issue with the license